bin/git-undo

```diff
@@ -1,6 +1,28 @@
 #!/usr/bin/env bash
 
 back="^"
+cstr="commits"
+ccnt=$(git rev-list --count HEAD)
+if [ $ccnt -eq 1 ]; then cstr="commit"; fi
+parm3=""
+
+function _undo()
+{
+  type=${1:-soft}
+  undo_cnt=${2:-1}
+  reset=${3:-""}
+echo "type=$type, cnt=$undo_cnt, reset=$reset"
```

Comment:
Please remove the debug log.

Reply:
Doh! I looked at that right before my last round of testing and said, "Don't forget to get rid of that!" and promptly didn't get rid of that.